### PR TITLE
Remove unused Python modules in test/test_sendgrid.py

### PR DESCRIPTION
**Pull request by gy741**
# Fixes # 

Hello,

In my opinion, No sys Python module and prism_cmd variable are required.

Due to the processing of annotations, the code does not work.

What do you think of it?

Thanks.

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [SendGrid](mailto:dx@sendgrid.com), or file a GitHub Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=712) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=712) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=712) it.</sub>

File: test/test_sendgrid.py [hunk 1]
```diff
@@ -24,7 +23,6 @@ def setUpClass(cls):
                 os.path.dirname(__file__)), '/..')
         cls.sg = sendgrid.SendGridAPIClient(host=host)
         cls.devnull = open(os.devnull, 'w')
-        prism_cmd = None
```
Comment (42B) [inline test/test_sendgrid.py]: It seems like both `import sys` and `prism_cmd` would be needed if that code below was ever uncommented though?

Comment (gy741) [reply test/test_sendgrid.py]: Hello @42B  Thank you for reviewing the code.
Yes, I agree with you.
In my opinion, it seems good to remove the annotated code for Readability.
Ref:https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

Comment (42B) [reply test/test_sendgrid.py]: @gy741 ,

Other than this one line, PR #653 seems to take care of this whole PR while also upgrading prism.

Comment (gy741) [reply test/test_sendgrid.py]: Oh, Yes exactly,
Except for one thing, the prism_cmd variable was not removed.

Comment (42B) [reply test/test_sendgrid.py]: @gy741 

> "Other than this one line..."

Comment (gy741) [reply test/test_sendgrid.py]: @42B Yes. :)
It would be nice to include it in PR #653.
 If you want, you will close me PR.

**Pull request closed without merging**